src/test/java/uk/gov/justice/digital/config/BaseSparkTest.java

```diff
@@ -42,4 +62,87 @@ public static void stopSession() {
 		spark.stop();
 	}
 
+	protected static void assertDeltaTableContainsPK(String tablePath, int primaryKey) {
```

Comment:
These were just moved from the duplicated version of this class (which is now deleted)